storageClient/files.go

```
@@ -0,0 +1,49 @@
+package storageClient
+
+import (
+	"fmt"
+	"io"
+	"net/http"
+	"os"
+)
+
+func (c *StorageClient) UploadFile(url string, encryptedFileName string, checksum string, size int) (int, error) {
+	file, err := os.Open(encryptedFileName)
+	if err != nil {
+		return 0, err
+	}
+	defer file.Close()
+	req, err := http.NewRequest("PUT", url, file)
+	if err != nil {
+		return 0, nil
```

do we want to return the error code?

---

yep! will do!